timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -128,20 +130,22 @@ public static TimeLockAgent create(
             MetricsManager metricsManager,
             TimeLockInstallConfiguration install,
             Refreshable<TimeLockRuntimeConfiguration> runtime,
```

Comment:
It's a bit of a shame this does _also_ contain ClusterConfigurations that may be wrong. I'm thinking as a defense mechanism, should we map any user-provided runtime config into a layer that rejects any attempts to access the ClusterConfiguration through this Refreshable?